src/utils/web3.js

```diff
@@ -60,6 +60,7 @@ export function truncateAddress(address) {
   return `${address.slice(0, 6)}...${address.slice(-4)}`
 }
 
+let web3
```

Comment:
I’m not sure if using the web3 or window.ethereum globals in this file work correctly in the first place, but this may well break inheriting the global web3.